Week2/MAKEME.md

```diff
@@ -51,10 +53,10 @@ class TodoModel {
 
 # Adding a new database user
 
-Until now we've always connected to the database as `root`. We don't want to
-allow our TODO app access to other databases than the TODO app itself:
+Until now we've always connected to the database as `root` (make sure you remember what a `root` user is). 
```

Comment:
This actually seems like it would be more confusing, since a database root user isn't the same as a system root user. Maybe there's another way to clarify.